piksi_tools/fileio.py

```diff
@@ -195,6 +259,8 @@ def get_args():
   """
   import argparse
   parser = argparse.ArgumentParser(description='Swift Nav File I/O Utility.')
```

Comment:
I'd love to remove the serial io command line option boilerplate and lift into one file:  Thoughts?
https://github.com/swift-nav/piksi_tools/blob/master/piksi_tools/serial_link.py#L54

Not mandatory and of course there shouldn't be skylark in their probably.